api/proto/teleport/legacy/types/types.proto

```diff
@@ -6876,6 +6891,9 @@ message PluginOktaSettings {
 
   // CredentialsInfo contains information about the Okta credentials.
   PluginOktaCredentialsInfo credentials_info = 5;
+
+  // EnableSiemIntegration enables SIEM (Security Information and Event Management).
+  bool enable_siem_integration = 6;
```

Comment:
```suggestion
  bool enable_system_log_export  = 6;
```

Okta names this system logs, should we name is as well? Do you have a better name?